config/highscoreCreatures.json

```diff
@@ -1,6 +1,6 @@
 {
     "creatures": [
-        { "min" : 1, "max" : 4, "creature": "imp" },
```

Comment:
That's also a functional change I think... Currently cheater (0 points) has no creature. I think that's better than "imp" for cheater.

Reply:
That's probably a separate logic. 
If creature is actually not found then it will result in a crash.
Besides - score of 0 is legal, if you play long enough.